packages/astro/src/@types/astro.ts

```diff
@@ -1017,6 +1017,7 @@ export interface AstroUserConfig {
 		 * @name markdown.drafts
 		 * @type {boolean}
 		 * @default `false`
+		 * @deprecated since version 3.0. Use content collections instead.
```

Comment:
@delucis - Is there any special formatting we need to adhere to for the `deprecated` entry here?

Reply:
We don’t actually display `@deprecated` comments in docs yet, so now’s our chance to decide a format if we want!

I can imagine generating something like this? Where the contents of the comments follow the leading `Deprecated:`.

<img width="784" alt="image" src="https://github.com/withastro/astro/assets/357379/eb712d6c-555e-4a5a-88a6-277e7654a186">

Or even something like this depending on how highlighted we want it:

<img width="826" alt="image" src="https://github.com/withastro/astro/assets/357379/526c0a80-33e7-48b5-a22b-2e9930b0e17d">

So given that, I guess it depends if you think specific wording would be clearest there @sarah11918?

Reply:
Actually, checking where I adopted the `since version X` pattern which is from `errors-data.ts`, it looks like we already render it today:

<img width="729" alt="image" src="https://github.com/withastro/astro/assets/34116392/75f47066-d861-4f5e-936c-1a1c4d3d0b3a">

So perhaps another place to align if we want to make `@deprecated` work nicely everywhere.

Reply:
Ah OK, yes, we could implement a similar pattern in the config reference if we like how the error generation script does this.

In this example I don’t love how basically a full sentence is split across heading and paragraph, so if we like the chunkier aside rendering, I would probably argue we should end up with something that looks more like:

> ⚠️ **Deprecated**
> Deprecated since version 2.6. Use content collections instead.

In which case, best for that to be reflected in the JSDoc comment too:

```suggestion
		 * @deprecated Deprecated since version 2.6. Use content collections instead.
```

Reply:
I pushed a commit to update the `@deprecated` tag to be a proper sentence instead 👍